docs/guides/ce-resource-management.rst

```diff
@@ -57,40 +57,46 @@ Convention is to use binary prefixes for memory resource requests (``disk`` and
 .. note:: mixing unit prefixes
     Binary and decimal units can be used interchangeably, however this can occasionally cause confusion, and care must be taken when specifying these parameters. For example, a memory request of ``100m`` specifies not 100 megabytes, but 100 millibytes, or 0.1 bytes.
 
-Setting Workflow Resources
---------------------------
+.. TODO: uncomment and check this section when workflow resource management is implemented (https://zapatacomputing.atlassian.net/browse/ORQSDK-797?atlOrigin=eyJpIjoiNGU1MDU0NjFhNTMxNGUwN2IyZTQzODMxZTVhNjQwM2UiLCJwIjoiaiJ9)
```

Comment:
I should have mentioned, I implemented this in between making the previous comment and now. That ticket is to do with heterogeneous workflows.

Reply:
[e150bda](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/96/commits/e150bdad7e314f7da95b1c319fd2d90be78360ac)